src/lib.rs

```diff
@@ -27,6 +27,7 @@ use crate::primitives::{
 };
 use crate::traits::{AuthoritySet, MmrState, StorageRead, StorageWrite};
 pub use beefy_primitives;
+pub use sp_core;
```

Comment:
any reason why?

Reply:
I didn't want to add  them as dependencies in ibc-rs.

Reply:
nahh, i think its fine